app/controllers/projects_controller.rb

```diff
@@ -477,4 +478,33 @@ def log_activity(type_of, project = nil, message_items = {})
             project: project,
             message_items: message_items)
   end
+
+  def set_breadcrumbs_items
+    project = @project
+    breadcrumbs_items = []
+    folders = helpers.tree_ordered_parent_folders(current_folder)
+    breadcrumbs_items.push({
+                 label: t('projects.index.breadcrumbs_root'),
+                 url: projects_path,
+                 class: 'project-folder-link'
+               })
+
+    folders&.each do |project_folder|
```

Comment:
Maybe for PET you can make some concern because this part same for all PET levels